### PR TITLE
Update styles

### LINKED ISSUE #412: Вся верстка летит!!! Дима, ты дурак?!?!??! Ишью объявляется открытым
Пишите доработки верстки сюда максимально точно. Старайтесь не повторяться
1. Исправить цвет фона.
2. Обновить лэндинг

### PR DESCRIPTION
**Pull request by deadit**
Fixes #412

### CODE REVIEW

File: services/app/assets/css/app.scss [hunk 1]
```diff
@@ -1,7 +1,15 @@
 $link-color:  #1f3754;
 $link-hover-color: #5673ce;
 $link-hover-decoration: none;
-$body-bg: #e9edf3;
+$body-bg: #fafafa;
+
+$gray-600:    #477591 !default;
```
Comment (puku) [inline services/app/assets/css/app.scss]: Not sure about code style, but think it's better to remove these spaces.

File: services/app/assets/js/widgets/containers/ChatWidget.jsx [hunk 2]
```diff
@@ -4,8 +4,8 @@ import { fetchState, addMessage } from '../middlewares/Chat';
 import { chatUsersSelector, chatMessagesSelector, currentChatUserSelector } from '../selectors';
 import Messages from '../components/Messages';
 import UserName from '../components/UserName';
-import InputWithEmoji from '../components/InputWithEmoji';
 
+const messageStyles = { wordBreak: 'break-all', height: '164px' };
```
Comment (puku) [inline services/app/assets/js/widgets/containers/ChatWidget.jsx]: Isn't it better to store all style in css files?

File: services/app/assets/js/widgets/containers/ChatWidget.jsx [hunk 3]
```diff
@@ -40,29 +41,30 @@ class ChatWidget extends React.Component {
...
+              className="form-control border-secondary"
+              placeholder="Type message here..."
               value={message}
-              handleChange={this.handleChange}
-              handleSubmit={this.handleSubmit}
+              onChange={this.handleChange}
             />
             <div className="input-group-append">
               <button className="btn btn-outline-secondary" type="button" onClick={this.handleSubmit}>
-                Send
```
Comment (puku) [inline services/app/assets/js/widgets/containers/ChatWidget.jsx]: It seems these spaces should be kept

File: services/app/assets/js/widgets/containers/GameList.jsx [hunk 4]
```diff
@@ -186,7 +186,7 @@ class GameList extends React.Component {
         aria-expanded="false"
       >
         <i className="fa fa-random mr-2" />
-        Play with any player
+        Create game
       </button>
```
Comment (vtm9) [inline services/app/assets/js/widgets/containers/GameList.jsx]: Create a game

File: services/app/assets/js/widgets/components/InputWithEmoji.jsx [hunk 5]
```diff
@@ -69,14 +69,14 @@ class InputWithEmoji extends PureComponent {
   }
 
   render() {
-    const { value = '' } = this.props;
+    const { inputVal } = this.state;
     return (
-      <>
-        <ContentEditable
+      <Fragment>
```
Comment (imamatory) [inline services/app/assets/js/widgets/components/InputWithEmoji.jsx]: Let's use `<></>` style Fragment

File: services/app/lib/codebattle_web/templates/page/landing.html.slim [hunk 6]
```diff
@@ -1,49 +1,39 @@
...
-            button.btn.btn-sm.btn-outline-success.m-2
-              i.icon-python.pr-1
-              = "Python"
-          a href="#{auth_path(@conn, :request, :github)}"
-            button.btn.btn-sm.btn-outline-success.m-2
-              i.icon-ruby.pr-1
-              = "Ruby"
+          a.m-2.btn.btn-sm.btn-outline-secondary href="#{auth_path(@conn, :request, :github)}"
+            i.icon-clojure.pr-1
+            = "Clojure"
```
Comment (imamatory) [inline services/app/lib/codebattle_web/templates/page/landing.html.slim]: Would be better to use loop for this, don't you think?

File: services/app/assets/js/widgets/components/InputWithEmoji.jsx (hunk 5 above)
Comment (deadit) [reply services/app/assets/js/widgets/components/InputWithEmoji.jsx]: Да я его убрал вообще ж

File: services/app/lib/codebattle_web/templates/page/landing.html.slim (hunk 6 above)
Comment (deadit) [reply services/app/lib/codebattle_web/templates/page/landing.html.slim]: I use copypasta pattern

Comment (deadit) [reply services/app/lib/codebattle_web/templates/page/landing.html.slim]: maybe latter, i will refactor this

File: services/app/assets/js/widgets/components/InputWithEmoji.jsx (hunk 5 above)
Comment (imamatory) [reply services/app/assets/js/widgets/components/InputWithEmoji.jsx]: So why it is green and marked with `+`?

File: services/app/lib/codebattle_web/templates/page/landing.html.slim (hunk 6 above)
Comment (deadit) [reply services/app/lib/codebattle_web/templates/page/landing.html.slim]: primi po bratski, brat

**Pull request merged**